app/cdap/components/WranglerV2/SearchField/index.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Box, Divider, Input, InputProps, Link, Typography } from '@material-ui/core';
+import CloseOutlinedIcon from '@material-ui/icons/CloseOutlined';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import T from 'i18n-react';
+import styled from 'styled-components';
+import { grey } from '@material-ui/core/colors';
+
+export interface IDirectiveSuggestionProps {
+  link: string;
+  usage: string;
+}
+
+interface ISearchFieldProps {
+  onPostfixIconClick: () => void;
+  inputProps: InputProps;
+  directiveSuggestion: IDirectiveSuggestionProps;
+}
+
+export const StyledInput = styled(Input)`
```

Comment:
instead of StyledInput use DirectiveInput or something specific - new comment from Sean

Reply:
Done